builtin/fstk/ui.lua

```diff
@@ -18,6 +18,7 @@
 ui = {}
 ui.childlist = {}
 ui.default = nil
+ui.override = false
```

Comment:
Please call this 


```suggestion
-- Whether fstk is currently showing its own formspec instead of active ui elements.
ui.overridden = false
```